api/app/src/main/kotlin/packit/controllers/PacketController.kt

```diff
@@ -41,15 +41,14 @@ class PacketController(private val packetService: PacketService)
         )
     }
 
-    @GetMapping("/packetGroupSummary")
-    fun getPacketGroupSummary(
+    @GetMapping("/packetGroupSummaries")
```

Comment:
This is one of several renamings I did while working on this - I found precise naming helped me think more accurately